app/liquid_tags/unified_embed/tag.rb

```diff
@@ -66,6 +70,12 @@ def self.validate_link!(link)
       raise StandardError, I18n.t("liquid_tags.unified_embed.tag.invalid_url")
     end
 
+    def self.handle_listings_disabled!(link)
+      return unless link.start_with?("#{URL.url}/listings/") && !Listing.feature_enabled?
```

Comment:
Pedantic question about an edge case, does the listings route always exist, or is that route masked when the feature is disabled (could I then create a page or user named listings?)

https://github.com/forem/forem/blob/main/config/routes.rb#L30-L34 
https://github.com/forem/forem/blob/main/config/initializers/reserved_words.rb#L109-L121
https://github.com/forem/forem/blob/main/config/application.rb#L88-L96

Or would a link to a user named "listings" be handled further up?

Reply:
Excellent question! I shall investigate...

Reply:
science project part done ("what is"), we _can_ create a user "listings" (whose path is probably masked when the listings feature gets turned back on and the route returns!).

```ruby
[6] pry(main)> u.username
=> "listings"
[7] pry(main)> u.valid?
=> true
```

"what should" is probably one more reserved word. Since listings had already always been a top level route until we made it optional, it's probably not going to impact anyone.